style.css

```diff
@@ -337,3 +337,65 @@ body {
   color: white;
   padding: 12px;
 }
+
+.text-container {
+  margin-top: 120px;
+  align-items: center;
+}
+
+.newprojects-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-form {
+  margin: 24px;
+}
```

Comment:
- [ ] Please use `flexbox` to align your items both in the `text-container` and in the `contact-form`, as it will help you later with responsivity. 😁

Reply:
I have implemented this